javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+const $off = document.querySelector('.circle-off');
```

Comment:
A better variable name would be `$circle`, since the circle could either be "on" or "off. By only referring to it as `$off`, it's kinda confusing why you'd turn it on.